.github/CODEOWNERS

```diff
@@ -17,13 +17,15 @@ contrib/debian @cosmix @Ferroin @prologic
 collectors/ @vlvkobal @mfundul @cosmix
 collectors/charts.d.plugin/ @ilyam8 @Ferroin @prologic @cosmix
 collectors/freebsd.plugin/ @vlvkobal @thiagoftsm @cosmix
+collectors/ebpf.plugin/ @thiagoftsm @vlvkobal @cosmix
 collectors/macos.plugin/ @vlvkobal @thiagoftsm @cosmix
 collectors/node.d.plugin/ @jacekkolasa @cosmix
 collectors/node.d.plugin/fronius/ @ccremer @cosmix
 collectors/node.d.plugin/snmp/ @jacekkolasa @cosmix
 collectors/node.d.plugin/stiebeleltron/ @ccremer @cosmix
 collectors/python.d.plugin/ @ilyam8
 collectors/cups.plugin/ @simonnagl @vlvkobal @thiagoftsm @cosmix
+exporting/  @vlvkobal @thiagoftsm
 daemon/ @thiagoftsm @mfundul @cosmix
 database/ @mfundul @thiagoftsm @cosmix
 docs/ @cosmix @joelhans @shortpatti
```

Comment:
add `/aclk

Reply:
Yes - `aclk/ @stelfrag @underhood @amoss`

Reply:
I added `aclk` with the last commit.